v2/client.go

```diff
@@ -423,6 +423,12 @@ func (c *Client) callAPI(ctx context.Context, r *request, opts ...RequestOption)
 	return data, nil
 }
 
+// SetApiEndpoint set api Endpoint
+func (c *Client) SetApiEndpoint(url string) *Client {
+	c.BaseURL = url
```

Comment:
Why not set API endpoint with `c.BaseURL = url`?

Reply:
one string futures.NewClient("", "").SetApiEndpoint("...")